### PR TITLE
WIP: use React Table for table panel

### PR DESCRIPTION
**Pull request by mtanda**
https://github.com/grafana/grafana/issues/12005

### CODE REVIEW

File: public/app/plugins/datasource/prometheus/datasource.ts [hunk 1]
```diff
@@ -42,6 +43,7 @@ export class PrometheusDatasource {
     this.basicAuth = instanceSettings.basicAuth;
     this.withCredentials = instanceSettings.withCredentials;
     this.interval = instanceSettings.jsonData.timeInterval || '15s';
+    this.queryTimeout = instanceSettings.jsonData.queryTimeout;
```
Comment (torkelo) [inline public/app/plugins/datasource/prometheus/datasource.ts]: this seems unrelated to the PR :)

Comment (mtanda) [reply public/app/plugins/datasource/prometheus/datasource.ts]: oops, I forgot to rebase...

**Comment by marefr** [comment]
This looks interesting. Just tried it out and it looks quite nice using the light theme. Dark theme styling missing. @torkelo have been started work with React panels so I'll assign him and hopefully he can give some valuable feedback.

**Comment by ryantxu** [comment]
@mtanda  -- check #15811

**Comment by mtanda** [comment]
@ryantxu Wow, thanks for taking over my work!!

Close my PR.
